Accepted-RFCs/RFC-013-new-initiative-process.md

```diff
@@ -0,0 +1,124 @@
+# RFC-013: New initiative process
+
+
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+
+Initially submitted on 24 Aug 2022
+
+## Current status
+
+- [x] Draft
+- [ ] Under discussion
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal overview
+
+The Good Docs Project has an issue of delivery: we haven't delivered a 1.0 of our templates since the inception. And as a volunteer org, we need to keep focus on our mission and goal. It's easy to dilute ourselves if we don't keep ourselves accountable to focus and intentional communal growth.
+
+In other words, we as a project need to make sure we can at least do our core mission well. All new initiatives should 
+
+(As a personal note, the drafter of this RFC has ADHD, so "new project energy" is a real thing, a true risk of distraction and effort dilution.)
+
+
+## Motivation
+
+As we grow, our risk of project scope creep and sprawl grows. As a group, we should grow *with intentionality*. 
+
+
+## Proposal
+
+When someone wants to start a new working group or sub-project in the Good Docs Project, it should coherently address at least one of:
+* We have an internal need, to make working on the project better—whether that's smoothing out processes, spinning up groups to keep work from being on one person, documenting information for worldwide efforts, etc
+* We identify some utility we can offer our userbase that either directly or reasonably indirectly builds off of our core mission: to delivery high-quality template resources to people who need help making their own documentation.
+
+Along with that, each new initiative needs someone to lead it. That person needs to be confident in leading (or co-leading) that initiative for at least a year. This is to prevent initiatives from being spun up and left to dwindle. (Unless said initiative isn't intended to last that long, of course. But we can expect some initiatives that intend to be short to take longer than initially planned.)
+
+Finally, each initiative needs to articulate the deliverables it intends to produce in its first year. This can of course mutate over time, but an initial understanding is key to helping the rest of the project understand the new initiative.
+
+With all of that, the initiative should be drafted as a RFC so the project steering committee can both ask insightful questions/provide comments, as flag their possible interest in helping with that initiative.
+
```

Comment:
Assuming the definition of "initiative" here is precisely "creating a new WG," this all looks logical. Just want to make sure there's a distinction between what's covered by this RFC and an "epic" (in release planning terms), which may be an ongoing goal that we never fully "complete," spans several releases, and is comprised on multiple tasks/issues.

I mention this because epics are a natural outcome of the backlog refinement process, and I don't think need an RFC or dedicated working group/leader.

Reply:
+1 to @flicstar 's decision tree.

suggestion: (non-blocking): Riffing further off that, how are we going to do the early brainstorming for ideas?
* How can someone say "I reckon this is a good idea, another one else want to join me?"
* Would it be worth introducing some sort of light prioritized "backlog" of ideas, so we can say to someone "Cool, you want to work on a GeeWiz template. You should talk to Jo who was also interested in that."
* Maybe also use the backlog to record a suggestion which was rejected and why, so we can reference it when the suggestion comes through again.
* I note that a backlog can become long and unwieldy, and so probably should be implemented with a way to easily ignore old suggestions. Maybe use  an archived email list.

Reply:
Revised this to be about new initiatives that require working groups.